### PR TITLE
Update Simplified Chinese Translation

### PR DESCRIPTION
**Pull request by sbzlzh**
@TheOnly8Z What do you think?What is your translation of 1864 and 1865?

### CODE REVIEW

File: lua/terrortown/lang/zh_hans.lua [hunk 1]
```diff
@@ -1846,23 +1846,23 @@ L.help_falldmg_exponent = [[
...
-
---L.tbut_adminarea = "ADMIN AREA:"
\ No newline at end of file
+L.testpopup_title = "一个测试弹出窗口，现在有一个多行标题，多好啊！"
+L.testpopup_subtitle = "好吧，你好！这是一个带有一些特殊信息的花式弹出窗口。文字也可以是多行的，多好啊！呃，如果我有什么想法的话，我可以添加这么多的文字..."
+
+L.hudeditor_chat_hint1 = "[TTT2][INFO] 将鼠标悬停在一个元素上，按住[LMB]并移动鼠标来移动或调整其大小。"
+L.hudeditor_chat_hint2 = "[TTT2][INFO] 按住ALT键进行对称调整大小。"
+L.hudeditor_chat_hint3 = "[TTT2][INFO] 按住SHIFT键，在轴上移动并保持长宽比。"
+L.hudeditor_chat_hint4 = "[TTT2][INFO] 按[RMB] -> 'Close'来退出HUD编辑器！"
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:1855]:
```suggestion
L.hudeditor_chat_hint4 = "[TTT2][INFO] 按 [RMB] -> 'Close' 来退出HUD编辑器！"
```

File: lua/terrortown/lang/zh_hans.lua [hunk 2]
```diff
@@ -1846,23 +1846,23 @@ L.help_falldmg_exponent = [[
...
+
+L.guide_nothing_title = "这里还没有消息!"
+L.guide_nothing_desc = "这是一项正在进行中的工作，通过在GitHub上为项目做贡献来帮助我们。"
+
+L.sb_rank_tooltip_developer = "TTT2开发者"
+L.sb_rank_tooltip_vip = "TTT2支持者"
+L.sb_rank_tooltip_addondev = "TTT2附加组件开发者"
+L.sb_rank_tooltip_admin = "服务器管理员"
+L.sb_rank_tooltip_streamer = "Streamer"
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:1864]:
```suggestion
L.sb_rank_tooltip_streamer = "主播"
```

File: lua/terrortown/lang/zh_hans.lua [hunk 3]
```diff
@@ -1846,23 +1846,23 @@ L.help_falldmg_exponent = [[
...
+L.sb_rank_tooltip_heroes = "TTT2 Heroes"
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:1865]:
```suggestion
L.sb_rank_tooltip_heroes = "TTT2 英雄"
```

File: lua/terrortown/lang/zh_hans.lua [hunk 4]
```diff
@@ -1846,23 +1846,23 @@ L.help_falldmg_exponent = [[
...
\ No newline at end of file
+L.testpopup_title = "一个测试弹出窗口，现在有一个多行标题，多好啊！"
+L.testpopup_subtitle = "好吧，你好！这是一个带有一些特殊信息的花式弹出窗口。文字也可以是多行的，多好啊！呃，如果我有什么想法的话，我可以添加这么多的文字..."
+
+L.hudeditor_chat_hint1 = "[TTT2][INFO] 将鼠标悬停在一个元素上，按住[LMB]并移动鼠标来移动或调整其大小。"
+L.hudeditor_chat_hint2 = "[TTT2][INFO] 按住ALT键进行对称调整大小。"
+L.hudeditor_chat_hint3 = "[TTT2][INFO] 按住SHIFT键，在轴上移动并保持长宽比。"
+L.hudeditor_chat_hint4 = "[TTT2][INFO] 按[RMB] -> 'Close'来退出HUD编辑器！"
+
+L.guide_nothing_title = "这里还没有消息!"
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:1857]:
```suggestion
L.guide_nothing_title = "这里暂时什么都没有！"
```

**Review by TheOnly8Z — CHANGES_REQUESTED** [review]

**Comment by TheOnly8Z** [comment]
Posted my recommended changes.

**Comment by sbzlzh** [comment]
all right

**Comment by sbzlzh** [comment]
Is there more?

**Comment by TheOnly8Z** [comment]
LGTM.

**Comment by sbzlzh** [comment]
ok

**Review by saibotk — APPROVED** [review]
Thank you :)

**Review by Histalek — APPROVED** [review]
:+1:

**Pull request merged**